src/ngraph_builder.cc

```diff
@@ -2030,6 +2030,34 @@ static Status TranslateRsqrtOp(
       });
 }
 
+static Status TranslateShapeOp(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map) {
+  shared_ptr<ng::Node> ng_input;
+  TF_RETURN_IF_ERROR(GetInputNodes(ng_op_map, op, &ng_input));
+  
+  // the shape of the input tensor which will be the value to the Constant Op
+  auto input_shape = ng_input->get_shape();
+  
+  // the rank of the input tensor which will be the shape to the Constant Op
+  auto rank = input_shape.size();
+
+  // the inputs to the Constant Op
+  // type
+  auto type = Builder::TF_NGRAPH_TYPE_MAP().at(DataType::DT_FLOAT);
```

Comment:
the // type could go in the above line, and format-check probably will still be happy

Reply:
DT_FLOAT -> DT_INT

Reply:
Oh, its fixed :-D